content/docs/dvclive/dvclive-with-dvc.md

```diff
@@ -38,36 +42,38 @@ Let's use `dvc stage add` to create a stage to wrap this code (don't forget to
 `dvc init` first):
 
 ```dvc
-$ dvc stage add -n train --live training_metrics
-                -d train.py python train.py
+$ dvc stage add \
+--name train \
+--deps train.py \
+--metrics-no-cache training_metrics.json \
+--plots-no-cache training_metrics/scalars \
+python train.py
 ```
 
-`dvc.yaml` will contain a new `train` stage with the DVCLive configuration (in
-the `live` field):
+<admon type="exclamation">
```

Comment:
No such type that I know of. Only info, tip, warn.

Reply:
It is one of the supported types:

https://github.com/iterative/dvc.org/blob/788b549dab46a2fddddf86180660a3c4348ba936/plugins/gatsby-theme-iterative-docs/src/components/Documentation/Markdown/Admonition/index.tsx#L5-L13

Reply:
I see! They only mentioned 3 in the description of https://github.com/iterative/cml.dev/pull/210. Maybe people have been adding more emojis but I'm still not sure that each type will have a different style (background color, etc.)

Reply:
Ya. It looks like it uses the same style as warn.

<img width="779" alt="image" src="https://user-images.githubusercontent.com/1477535/166838869-aa5e6285-b2c9-490c-bafb-f70d3f2d560c.png">
> See https://dvcorg-deprecatelive.gtsb.io/doc/dvclive/dvclive-with-dvc

Reply:
We only use info, warn, tip in the codebase so far. I think we should stick to those if possible. I'll check with the web team on why there's so many types.

```suggestion
<admon type="warn">
```

Reply:
> It is one of the supported types:

Those aren't actually types. There are only three type options:

1. info (creates purple and default emoji is ℹ️)
2. tip (creates blue and default emoji is 💡)
3. warn (creates orange and default emoji is ⚠️)

Those are `icon` options! Which you can use in combination with `type`! Since you want an exclamation `icon`: 
```suggestion
<admon type="warn" icon="exclamation">
```

Reply:
Thanks for clarifying @julieg18 !

So up to you @daavoo but I vote for plain `warn`.